dockerutil/image.go

```diff
@@ -253,3 +262,7 @@ func DefaultLogImagePullFn(log buildlog.Logger) func(ImagePullEvent) error {
 		return nil
 	}
 }
+
+func isTLSVerificationErr(err error) bool {
+	return err != nil && strings.Contains(err.Error(), "tls: failed to verify certificate: x509: certificate signed by unknown authority")
```

Comment:
Not necessarily, any errors related to TLS failing will still ultimately cause envbox to fail, I added this specifically to avoid one of the tests spinning for an extra 30s unnecessarily.